tests/php/bootstrap.php

```diff
@@ -59,10 +59,13 @@ function amp_unit_test_load_plugin_file() {
  */
 define( 'WP_CLI_ROOT', TESTS_PLUGIN_DIR . '/vendor/wp-cli/wp-cli' );
 define( 'WP_CLI_VENDOR_DIR', TESTS_PLUGIN_DIR . '/vendor' );
-require_once WP_CLI_ROOT . '/php/utils.php';
 
-$logger = new WP_CLI\Loggers\Regular( true );
-WP_CLI::set_logger( $logger );
+if ( file_exists( WP_CLI_ROOT . '/php/utils.php' ) ) {
```

Comment:
Why was the `file_exists()` check needed? Is it due to WP-CLI not being able to be installed on PHP 8.2?

Reply:
Not 100% sure right now (not at computer) but yeah I think it was because of issues installing it. Maybe because of Requests? Will have to check it out again.

Reply:
Here's the run before that change: https://github.com/ampproject/amp-wp/actions/runs/2925960305/jobs/4667211482

```
PHP Warning:  require_once(/tmp/wordpress/src/wp-content/plugins/amp/vendor/wp-cli/wp-cli/php/utils.php): Failed to open stream: No such file or directory in /tmp/wordpress/src/wp-content/plugins/amp/tests/php/bootstrap.php on line 62
```